### DISCUSSION TITLE
Welcome to Resume Chatbot discussions!

### DISCUSSION THREAD

**Discussion by romankhar** [egen/resume-chatbot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GodApollo7** [comment]
Hello can your project rank the resume according to the need of the job?

**Reply by romankhar** [reply]
This was the whole idea of the project. In the current form it will select sorted top resumes that are most fit for the job, given the job description. However I have not done qualitative testing to verify the accuracy of the results.
